lib/browser/rpc-server.js

```diff
@@ -146,6 +146,28 @@ const throwRPCError = function (message) {
   throw error
 }
 
+const rendererMissingErrorMessage = (meta, args, callIntoRenderer) => {
+  let message = `Attempting to call a function in a renderer window that has been closed or released.` +
+    `\nFunction provided here: ${meta.location}`
+
+  if (args.length === 0) return message
+  if (!args[0].sender || !args[0].sender.eventNames) return message
+
+  const sender = args[0].sender
+  const remoteEvents = sender.eventNames().filter((eventName) => {
+    return sender.listeners(eventName).includes(callIntoRenderer)
+  })
+
+  if (remoteEvents.length > 0) {
+    message += `\nRemote event names: ${remoteEvents.join(', ')}`
+    remoteEvents.forEach((eventName) => {
+      sender.removeListener(eventName, callIntoRenderer)
```

Comment:
Would be great to assert this happens as well in the spec, perhaps by asserting `listenerCount()` goes down for each event name being listened to.